docs/installation/cloud.md

```diff
@@ -0,0 +1,64 @@
+<!--[metadata]>
++++
+title = "Installation on a cloud platform"
+description = "Installation instructions for Docker on Amazon EC2."
+keywords = ["cloud, docker, machine, documentation,  installation"]
+[menu.main]
+parent = "smn_cloud"
++++
+<![end-metadata]-->
+
+# Cloud providers
+
+You can install Docker on any cloud platform that runs an operating system (OS) that Docker supports. This includes many flavors and versions of Linux, along with Mac and Windows.
+
+__REVIEWERS NOTE:__ The plan is to remove all the detailed info on each cloud provider that's currently under the "cloud install" topic (https://docs.docker.com/machine/drivers/). Are the following very general steps enough to replace that with?
```

Comment:
Alternatively, you can either put these notes in the description of your pull request or, if extra attention is needed on a specific part of your pull request, add an inline note on GitHub (like this comment I left here). Doing so prevents these notes from accidentally being merged :smile: